ancar/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,20 +41,33 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       PreparedQuery results = datastore.prepare(query);
 
       List<String> comments = new ArrayList<>();
+      List<String> commentsToPrint;
 
       for (Entity entity : results.asIterable()) {
         String comment = (String) entity.getProperty("comment");
         comments.add(comment);
       }
 
-      Gson gson = new Gson();
-
       String valueParameter = request.getParameter("value");
       int numberOfComments = Integer.parseInt(valueParameter);
```

Comment:
No need to declare a variable here,
```java
int numberOfComments = Integer.parseInt(request.getParameter("value"));
``` 
would be also fine

Reply:
should I resolve all of your suggestions and create a new pull request?